src/targets/Logary.Targets.Stackdriver/Target_Stackdriver.fs

```diff
@@ -0,0 +1,173 @@
+﻿// DOCUMENT YOUR MODULE
+module Logary.Targets.Stackdriver
+
+open Hopac
+open Hopac.Infixes
+open Logary
+open Logary.Target
+open Logary.Internals
+open Google.Logging.V2
+open Google.Api
+open Google.Logging.Type
+open Logary.Utils.Chiron
+open Logary.Utils.Chiron.Builder
+
+/// A structure that enforces certain labeling invariants around monitored resources
+/// TODO: add more resource types.
```

Comment:
I think that the vast majority of resource types will be either Compute or Container, with maybe a smattering of AppEngine. It's not hard to add additional types, I just am not 100% sure on what makes a viable first version of the target.